### DISCUSSION TITLE
NHL channel .m3u8 or .m3u

### DISCUSSION THREAD

**Discussion by jinx5** [iptv-org/community#982] [Q&A]
Any .m3u8 or .m3u that has the NHL game? 
Me and the buddy wants to know. 
Something to put on the box. 
I couldn't find NHL at all.

**Comment by ThetaX55** [comment]
Try some north American country channels. 
Found a channel. That had a major game. 
Don't expect to get all the games. there are to many.
If you got cable TV there should be a channel. or Antenna. There are payed apps on the smart TV. If your a hardcore fan. I don't care to much to go that far in paying. Maybe a sports bar would.

**Comment by freearhey** [comment]
If the request is still relevant, please repost it using this [form](https://github.com/iptv-org/iptv/issues/new?assignees=&labels=channel+search&projects=&template=4_channel-search.yml&title=Find%3A+).
